frontend/src/metabase/containers/UndoListing.jsx

```diff
@@ -62,49 +54,57 @@ UndoToast.propTypes = {
 };
 
 function UndoToast({ undo, onUndo, onDismiss }) {
-  const style = undo.toastColor
-    ? { backgroundColor: color(undo.toastColor) }
-    : undefined;
   return (
-    <ToastCard dark data-testid="toast-undo" style={style}>
-      <CardContent>
-        <CardContentSide>
-          <CardIcon name={undo.icon || "check"} color="white" />
-          {renderMessage(undo)}
-        </CardContentSide>
-        <CardContentSide>
-          {undo.actions?.length > 0 && (
-            <UndoButton onClick={onUndo}>{t`Undo`}</UndoButton>
-          )}
-          <DismissIcon name="close" onClick={onDismiss} />
-        </CardContentSide>
-      </CardContent>
-    </ToastCard>
+    <Motion
```

Comment:
Previously this component had no animation. This was fine in the old design, but in the new design the `BulkActions` toast component is animated while entering the screen. It would look jarring if it was then immediately replaced by the `UndoListing` toast without an entrance animation, so I added one.